util/util.go

```diff
@@ -30,6 +32,54 @@ func StrToSemVer(version string) (*semver.Version, error) {
 	return v, nil
 }
 
+func SemVerMatchRange(k8sVersion, versionRange string) (bool, error) {
+	toMatch, err := sv.Make(k8sVersion[1:])
+	if err != nil {
+		return false, fmt.Errorf("Cluster version [%s] can not be parsed as semver: %v", k8sVersion, err)
+	}
+	semVerRange, err := sv.ParseRange(versionRange)
+	if err != nil {
+		return false, fmt.Errorf("Failed to parse semver range [%s]: %v", versionRange, err)
+	}
+	if semVerRange(toMatch) {
+		logrus.Debugf("SemVerMatchRange: Cluster version [%s] matches range [%s]", k8sVersion, versionRange)
+		return true, nil
+	}
+	return false, nil
+}
+
+func RemoveZFromBinds(binds []string) []string {
+	var returnSlice []string
+	for _, element := range binds {
+		lastIndex := strings.LastIndex(element, ":")
+		bindOptions := element[lastIndex+1:]
+
+		if bindOptions != "" {
+			var cleanBindOptions []string
+
+			splitBindOptions := strings.Split(bindOptions, ",")
+			if len(splitBindOptions) >= 1 {
+				for _, bindOption := range splitBindOptions {
+					if strings.EqualFold(bindOption, "z") {
+						continue
+					}
+					cleanBindOptions = append(cleanBindOptions, bindOption)
+
+				}
+			}
+			var newString string
+			if len(cleanBindOptions) > 0 {
+				newString = fmt.Sprintf("%s%s", element[:lastIndex], fmt.Sprintf(":%s", strings.Join(cleanBindOptions, ",")))
+			} else {
+				newString = fmt.Sprintf("%s%s", element[:lastIndex], fmt.Sprintf("%s", strings.Join(cleanBindOptions, ",")))
```

Comment:
I do not believe we need the `Sprintf` here. Could be wrong.
```go
fmt.Sprintf("%s%s", element[:lastIndex], fmt.Sprintf("%s", strings.Join(cleanBindOptions, ",")))

# could be...

fmt.Sprintf("%s%s", element[:lastIndex], strings.Join(cleanBindOptions, ","))
```

Reply:
True, this came from a previous attempt where I need to concat the joined value with something else and then no longer needed it.